### ISSUE TITLE
libretube services Download Service stop

### ISSUE THREAD

**Issue by ghost** [libre-tube/LibreTube#4279]
### Steps to reproduce

com.github.libretube.services.DownloadService$stop$1$1.invokeSuspend(SourceFile:61)
	

### Expected behavior

Expected working well

### Actual behavior

java.lang.NullPointerException: Parameter specified as non-null is null: method com.github.libretube.services.DownloadService$stop$1$1.invokeSuspend, parameter <this>
	at com.github.libretube.services.DownloadService$stop$1$1.invokeSuspend(SourceFile:61)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(SourceFile:9)
	at kotlinx.coroutines.DispatchedTask.run(SourceFile:116)
	at android.os.Handler.handleCallback(Handler.java:907)
	at android.os.Handler.dispatchMessage(Handler.java:105)
	at android.os.Looper.loop(Looper.java:216)
	at android.app.ActivityThread.main(ActivityThread.java:7586)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1034)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@e37151c, Dispatchers.Main.immediate]
    

### LibreTube version

0.16.1

### Android version

Android 9

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by akano12** [comment]
Is it hard to follow the correct issue format?

**Comment by Bnyro** [comment]
Account already deleted, interesting.

**Comment by Bnyro** [comment]
Closing due to no way to reproduce.
